optimism/src/mips/proof.rs

```diff
@@ -19,13 +19,15 @@ use rayon::iter::{
     IntoParallelRefMutIterator, ParallelIterator,
 };
 
+/// Represents one line of the execution trace of the virtual machine
 #[derive(Debug)]
 pub struct WitnessColumns<G> {
     pub scratch: [G; crate::mips::witness::SCRATCH_SIZE],
     pub instruction_counter: G,
     pub error: G,
 }
 
+/// Helpers to iterate over the elements of the execution trace in parallel
```

Comment:
Not sure that this adds any value; I'd lean towards closing this to keep noise down.

Reply:
Removed